datasets/offenseval2020_tr/offenseval2020_tr.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# Lint as: python3
+"""OffensEval-TR 2020: A Corpus of Turkish Offensive Language on Social Media"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import logging
+import os
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{coltekin2020lrec,
+ author  = {Cagri Coltekin},
+ year  = {2020},
+ title  = {A Corpus of Turkish Offensive Language on Social Media},
+ booktitle  = {Proceedings of The 12th Language Resources and Evaluation Conference},
+ pages  = {6174--6184},
+ address  = {Marseille, France},
+ url  = {https://www.aclweb.org/anthology/2020.lrec-1.758},
+}
+"""
+
+
+_DESCRIPTION = """\
+OffensEval-TR 2020 is a Turkish offensive language corpus. The corpus consist of randomly sampled tweets and annotated in a similar way to OffensEval and GermEval.
+"""
+
+_HOMEPAGE = "https://coltekin.github.io/offensive-turkish/"
+_DOWNLOAD_URL = "https://coltekin.github.io/offensive-turkish/offenseval2020-turkish.zip"
+_FOLDER_NAME = "offenseval-tr-{split}-v1"
+
+
+class OffensEval2020TRConfig(datasets.BuilderConfig):
+    """BuilderConfig for OffensEval2020TR."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for OffensEval2020TR.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(OffensEval2020TRConfig, self).__init__(**kwargs)
+
+
+class OffensEval2020TR(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Offenseval2020TR(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `offenseval2020_tr`